ch_util/rfi.py

```diff
@@ -411,9 +422,16 @@ def frequency_mask(freq_centre, freq_width=None):
     freq_start = freq_centre - freq_width / 2
     freq_end = freq_centre + freq_width / 2
 
-    for fs, fe in bad_frequencies:
-        tm = np.logical_and(freq_end > fs, freq_start < fe)
-        mask = np.logical_or(mask, tm)
+    # Time-dependent static RFI flagging
+
+    for line in bad_frequencies:
+        fs, fe = line[1]
+
+        if not line[0] or timestamp >= line[0]:
+            tm = np.logical_and(freq_end > fs, freq_start < fe)
+            mask = np.logical_or(mask, tm)
+        else:
+            continue
```

Comment:
You can use Python's unpacking to stream line this
```suggestion
    for start_time, (fs, fe) in bad_frequencies:
        if start_time is None or timestamp >= start_time:
            tm = np.logical_and(freq_end > fs, freq_start < fe)
            mask = np.logical_or(mask, tm)
        else:
            continue
```
Also it's usually better to explicitly check for `None`, as for instance `start_time = 0.0` would give `True` for `not start_time`